### ISSUE TITLE
DiffPaneModel could report pass/fail

### ISSUE THREAD

**Issue by daveyostcom** [mmanela/diffplex#58]
I'm using DiffPlex to show differences when a unit test fails. Thank you!

For such uses, a property like this would be convenient in `DiffPaneModel`:
```
bool HasDifferences { get; }
```

Workaround is to first check if the strings are equal and call diff only if they're not.

**Comment by mmanela** [comment]
Interesting suggestion, would you be interesting in contributing?

**Comment by daveyostcom** [comment]
Sorry, can't.

**Comment by michasacuer** [comment]
Have same issue as @daveyostcom . That property would be amazing. @mmanela  i can contribute if u wish :) Do you have any slack or other for this project?

**Comment by mmanela** [comment]
@michasacuer I don't have a slack but we can use this item to discuss if that works

**Comment by michasacuer** [comment]
Cool, if i will have questions i will post it here :)

**Comment by michasacuer** [comment]
Im allowd to use Linq?

**Comment by daveyostcom** [comment]
Erik Meijer when asked what features of C# aren't used enough, answered LINQ and async/await. That was a while ago. 😎

**Comment by mmanela** [comment]
Linq is great!

**Comment by daveyostcom** [comment]
While we're on the subject of diffing actual vs expected in test results...

Whereas +/–/␠ and green/red/grey work well to show new/old/same versions, when they are used to indicate actual vs expected test results, they are not so successful because they incur a cognitive load. Quick:
- Do +/– mean good/bad?
- Do +/– mean actual/expected or expected/actual?
- Should I think of actual/expected as old/new or new/old?
- Red goes with – and green goes with +; does red mean bad and green mean good?
```
  0 0.0
- 1 1.0
+ 1 0.1
  2 2.0
- 3 3.0
  4 4.0
  5 4.1
+ 6 4.2
```
Instead of + and –, it occurred to me to use
- √ and green for expected (good, but missing in the actual result)
- x and red for actual (bad, added or changed in the actual result)

I think this solution removes the cognitive load nicely.
```
  0 0.0
√ 1 1.0
x 1 0.1
  2 2.0
√ 3 3.0
  4 4.0
  5 4.1
x 6 4.2
```
I couldn't find a better Unicode checkmark that works reliably in monospaced fonts.

**Comment by michasacuer** [comment]
@mmanela  i have a question, when you push new release on nuget (with `has differences`)?

**Comment by mmanela** [comment]
@michasacuer I just published 1.6.2
